### DISCUSSION TITLE
Sending multiple parameters in contract call

### DISCUSSION THREAD

**Discussion by kriptoe** [wevm/wagmi#2055] [General]
How do I send 2 or more parameters to a read function call. The code below isnt working

  const { data: nftTokens, error: getNFTError } = useContractRead({
    ...contractConfig,
    functionName: "tokenOfOwnerByIndex",
    args: ["0x2a21Cc5D8Bcaa0D10078C99606B03Ee46C58817d", 0], //hardcoded address can create a state variable
  });

**Comment by jxom** [comment]
Can you provide a minimal reproducible example via https://new.wagmi.sh so we can take a look?

**Comment by kriptoe** [comment]
Its ok, I had the wrong abi connected, works fine thanks
